frontend/src/app/components/master-page/master-page.component.html

```diff
@@ -62,6 +62,21 @@ <h6 *ngIf="env.LIQUID_ENABLED || env.BISQ_ENABLED" class="dropdown-header" i18n=
 </nav>
 </header>
 
+<div *ngIf="true || network.val === 'testnet' || network.val === 'signet'">
```

Comment:
add liquidtestnet

Reply:
Fixed in https://github.com/mempool/mempool/commit/d54aed376d0fbcee770bc4aa1e5c00c65c658ccb